openproblems/tasks/_batch_integration/_common/methods/baseline.py

```diff
@@ -0,0 +1,76 @@
+from .....tools.decorators import method
+from .....tools.utils import check_version
+from ...batch_integration_graph.methods.baseline import _randomize_features
+from ...batch_integration_graph.methods.baseline import _randomize_graph
+from ...batch_integration_graph.methods.baseline import _set_uns
+
+import functools
+
+_baseline_method = functools.partial(
+    method,
+    paper_name="Open Problems for Single Cell Analysis",
+    paper_reference="openproblems",
+    paper_year=2022,
+    code_url="https://github.com/openproblems-bio/openproblems",
+    is_baseline=True,
+)
+
+
+@_baseline_method(
+    method_name="No Integration",
+)
+def no_integration(adata, test=False):
```

Comment:
Is this redefined in bi_graph? It should be imported there from _common too rather than defined twice.